docs/quick-start.md

```diff
@@ -18,22 +18,33 @@ In this example, the train data set has numbers ranging from 1-10. Whereas in th
 import numpy as np
 from numalogic.models.autoencoder import AutoencoderPipeline
 from numalogic.models.autoencoder.variants import Conv1dAE
+from numalogic.models.threshold._std import StdDevThreshold
 from numalogic.postprocess import tanh_norm
+from numalogic.preprocess.transformer import LogTransformer
 
 X_train = np.array([1, 3, 5, 2, 5, 1, 4, 5, 1, 4, 5, 8, 9, 1, 2, 4, 5, 1, 3]).reshape(-1, 1)
 X_test = np.array([-20, 3, 5, 40, 5, 10, 4, 5, 100]).reshape(-1, 1)
 
-model = AutoencoderPipeline(
-   model=Conv1dAE(in_channels=1, enc_channels=4), seq_len=8, num_epochs=30
+# preprocess step
+clf = LogTransformer()
+train_data = clf.fit_transform(X_train)
+test_data = clf.transform(X_test)
+
+# Define threshold estimator and call fit()
+thresh_clf = StdDevThreshold(std_factor=1.2)
+thresh_clf.fit(train_data)
+
+ae_pl = AutoencoderPipeline(
+    model=Conv1dAE(in_channels=1, enc_channels=4), seq_len=8, num_epochs=30
 )
 # fit method trains the model on train data set
-model.fit(X_train)
+ae_pl.fit(X_train)
 
 # predict method returns the reconstruction error
-recon = model.predict(X_test)
+anomaly_score = ae_pl.score(X_test)
```

Comment:
Please let's use `predict`

Reply:
Basically my concern is that the predict function can give out a negative value but score will be always positive because it is recon error. And I believe our final anomaly score should be positive.